lib/Graph/Graph.cpp

```diff
@@ -1492,15 +1492,21 @@ Function::createFusedRowwiseQuantizedSparseLengthsWeightedSum(
       name, outTy, data, weights, indices, lengths));
 }
 
-FusedRowwiseQuantizedSparseLengthsWeightedSumNode *
+FusedRowwiseQuantizedSparseLengthsSumNode *
 Function::createFusedRowwiseQuantizedSparseLengthsSum(llvm::StringRef name,
                                                       Constant *data,
                                                       NodeValue indices,
                                                       NodeValue lengths) {
-  auto ty = getParent()->uniqueType(ElemKind::FloatTy, {indices.dims()[0]});
-  auto ones = createSplat(name.str() + ".ones", ty, 1.0);
-  return createFusedRowwiseQuantizedSparseLengthsWeightedSum(name, data, ones,
-                                                             indices, lengths);
+  auto inDims = data->dims();
```

Comment:
This logic is just copied from `createFusedRowwiseQuantizedSparseLengthsWeightedSum()` right? Can we just add a static helper here called by both that returns the `TypeRef`?

Reply:
Done.